packages/breadcrumbs/index.js

```diff
@@ -1,20 +1,25 @@
-import { FabricWebComponent } from '../utils';
+import { html } from 'lit';
+import { FabricElement } from '../utils';
 
-class FabricBreadcrumbs extends FabricWebComponent {
-  connectedCallback() {
-    const children = Array.from(this.children)
-      .map((child) => child.outerHTML)
-      .join('<span class="select-none" aria-hidden="true">/</span>');
+const separator = html`<span class="select-none" aria-hidden="true">/</span>`;
+const interleave = (arr) =>
+  [].concat(...arr.map((el) => [el, separator])).slice(0, -1);
 
-    this.shadowRoot.innerHTML += `<nav
-            aria-label="Her er du"
-            class="flex space-x-8"
-        >
-            <h2 class="sr-only">Her er du</h2>
-            ${children}
-        </nav>`;
+class FabricBreadcrumbs extends FabricElement {
+  connectedCallback() {
+    super.connectedCallback();
+    // Grab existing children at the point that the component is added to the page
+    // Interleave "/" separator with breadcrumbs
+    this._children = interleave(Array.from(this.children));
+  }
 
-    this.innerHTML = '';
+  render() {
+    return html`
+      <nav aria-label="Her er du" class="flex space-x-8">
+        <h2 class="sr-only">Her er du</h2>
+        ${this._children}
+      </nav>
+    `;
```

Comment:
The render method ignores the original children and instead renders out the interleaved ones stored in this._children